Meghna/README.md

```diff
@@ -0,0 +1,661 @@
+
+
+
+**Downloading Programs**
+
+Let's start by downloading the following programs on your computer:
+
+- Bot Framework Composer: [Windows](https://aka.ms/bf-composer-download-win) | [macOS](https://aka.ms/bf-composer-download-mac) | [Linux](https://aka.ms/bf-composer-download-linux)
+- Bot Framework Emulator: https://github.com/Microsoft/BotFramework-Emulator/releases
+- Postman: https://www.postman.com/downloads/
+- SQL Server Management Studio: https://aka.ms/ssmsfullsetup
+- Visual Studio: https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16 
+
+
+
+**Creating An Account With Microsoft Azure**
+
+Once you have installed all five programs, create an account on Microsoft Azure if you have not already. If you have an account with Microsoft Azure, feel free to skip this step. 
+
+1. Go to https://azure.microsoft.com/free
+2. Select the `Start free` option. 
+3. Sign in using either your Microsoft or GitHub account to create an account on Azure.
+4. Once it redirects you to the `About you` page, choose your country/region. Then, enter your first and last name, email address, and phone number. Select `Next`.
+5. On the `Identity verification by phone` page, choose your country code, and type in your phone number. Enter the verification code sent to you and select `Verify code`.
+6. If the verification code is correct, it will prompt you to enter your credit card details. When done, select `Next`.
+7. Review the agreement and privacy statement and select `Sign up`.
+
+
+
+**Creating New Folders To Store Files** 
+
+When you are all signed up for Azure, the next step is to create a few folders on your computer. 
+
+1. Go to the local disk on your computer.
+
+2. Create a folder. For our purposes, we will name the folder `Scheduling Bot`, but feel free to enter any name you prefer.
+
+3. Once you create a folder, create three other folders within the first one. Feel free to name it anything you like, but for our purposes, we will call it the following: 
+
+   `SchedulerBot (1)`
+
+   `SchedulerBotWebApp (2)`
+
+   `Scheduler Function (3)`
+
+   ![](Screenshots/Screenshot%20(6).png)
+
+
+
+**Composing The Bot (Stage 1)**
+
+We must now start composing the scheduler bot using the Bot Framework Composer. 
+
+*Side note: The Bot that we are creating will not have any advanced features, such as intents or Triggers, as it will only extend the `WelcomeTheUser` activity. Regardless, it gets the main job done, and is a cool program to create! 
```

Comment:
```suggestion
*Side note: The Bot that we are creating will not have any advanced features, such as intents or Triggers, as it will only extend the `WelcomeTheUser` activity. Regardless, it gets the job done, is easy to extend, and is a cool program to create! 
```